package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@markdoc/markdoc",
   "author": "Ryan Paul",
-  "version": "0.3.5",
+  "version": "0.3.6",
```

Comment:
We should probably release this as a new minor version right? Just in case it breaks folks' validation.